packages/cmake/project.bri

```diff
@@ -184,7 +185,9 @@ export function cmakeBuild(
   }
 
   return std.runBash`
-    export LIB="$LIBRARY_PATH"
```

Comment:
I removed this environment variable, since CMake shouldn't use it as is. But maybe I'm missing something here. Do you remember why we were having this env var @kylewlacy ? It was coming from this [PR](https://github.com/brioche-dev/brioche-packages/pull/144).